deployments/vanilla/terraform/main.tf

```diff
@@ -187,7 +187,7 @@ module "kubeflow_components" {
 #---------------------------------------------------------------
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "3.14.4"
+  version = "5.0.0"
```

Comment:
Thanks for the PR, Could you update other stacks as well?
https://github.com/search?q=repo%3Aawslabs%2Fkubeflow-manifests%20terraform-aws-modules&type=code